flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
Given it's Java, is it actually working only on x86_64?

Reply:
The upstream only have this. And it's not possible to pull dependencies for java in flatpak in build step

Reply:
I don't mean building from source. If there's no architecture-specific files in the deb, it can be used also on aarch64. @hfiguiere is it something you could test?

Reply:
just check the content of the flatpak. If it installed from the .deb any .so or binaries, then it's unlikely to work.

it should be possible to build from source (even with dependencies that you need to pull into the manifest).

Reply:
so i think including aarch64 is not feasible so far